.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
 .vscode/
+.history/
```

Comment:
Why was this added? I don't think the repository needs this, you can keep it in your local and not commit it to the repository.

Reply:
this dir  is a vscode extension local history use This is a directory used by a vscode or idea plugin called local history, which I can remove.